kong/runloop/plugin_servers/pb_rpc.lua

```diff
@@ -393,13 +393,10 @@ function Rpc:handle_event(plugin_name, conf, phase)
     instance_id = instance_id,
     event_name = phase,
   }, true)
-  if not res then
+  if not res or res == "" then
```

Comment:
seems `res` can be either `nil` or a table, wondering why include `res == ""` here

Reply:
lua_protobuf has changed how it manages default values between versions.  there are settings, but this is a safe solution too.